packages/scalable-data-sync/src/probabilities.ts

```diff
@@ -0,0 +1,113 @@
+type TErrorForK = Float32Array;
+type TAllErrorRates = Array<TErrorForK>;
+
+// prettier-ignore
+export const kErrors: TAllErrorRates = [
```

Comment:
what are these?

Reply:
@danisharora099 https://github.com/waku-org/nim-sds/blob/master/src/private/probabilities.nim

Reply:
I added comments explaining the values and how they are used in the bloom filter. Basically, these constants are used to determine what parameters to use in constructing a bloom filter with a false-positive rate below a certain threshold. (A lookup in a bloom filter will never report false-negatives, but there is always a chance of false-positives)